packages/salesforcedx-vscode-core/package.json

```diff
@@ -86,6 +86,7 @@
       "main": "dist/index.js",
       "dependencies": {
         "applicationinsights": "1.0.7",
+        "@heroku/functions-core": "0.4.2",
         "@salesforce/core": "3.30.9",
```

Comment:
Add functions-core as a module to be installed for the packaging.  The size increase was pretty minimal I think, but will verify.

Reply:
Curious what the difference was here?

Reply:
previously the code was functions core was included in our 'bundle' 
This was the root of the bug as they depend on being able to read a templates directory in their package node module and it didn't exist after bundling.  I can show you if this makes no sense.

Reply:
also I opened a bug in that repo about them not being 'bundle-able'.  We'll see if it can pick up some traction

Reply:
Before: 
`Uploading /home/circleci/project/extensions/salesforcedx-vscode-core-56.4.0.vsix (24 MB): DONE`
After: 
Uploading /home/circleci/project/extensions/salesforcedx-vscode-core-56.4.0.vsix (29 MB): DONE
5 MB diff or a 20% size increase 😢 Not as trivial as I'd hoped